static/docs/user-guide/running-dvc-on-windows.md

```diff
@@ -5,6 +5,13 @@ They are mostly related to NTFS (a file system that the Windows uses)
 characteristics and Windows built-in security mechanisms. Below are some
 workarounds that can help avoid these performance penalties:
 
+## POSIX-like command line terminal
```

Comment:
have you tried to run the get started with those? It's still not clear what was the deal with that issue on Windows.

Reply:
I have not ( since I don't have a Windows machine. Do you have a Windows Server that I can [RDP](https://www.microsoft.com/en-us/p/microsoft-remote-desktop/9wzdncrfj3ps) into perhaps?

And do you mean this issue? #490  There's also #134in this repo
and https://github.com/iterative/dvc/issues/2294, https://github.com/iterative/dvc/issues/2059 in the core repo.

Reply:
I have not ( since I don't have a Windows machine. Do you have a Windows Server that I can [RDP](https://www.microsoft.com/en-us/p/microsoft-remote-desktop/9wzdncrfj3ps) into perhaps?

And do you mean this issue? #490  There's also #134in this repo
and https://github.com/iterative/dvc/issues/2294, https://github.com/iterative/dvc/issues/2059 in the core repo.

Reply:
p.s. I meant "...do you mean this issue? #468 There's also #134 in this repo..."

Reply:
Ping @shcheklein

Reply:
the issue I meant is #486 . If we do this section, I would say we need to adjust the text above a little bit, because it's mostly performance focused.

Reply:
PR #486 (merged) addressed issue #395 (closed) by creatind https://dvc.org/doc/user-guide/running-dvc-on-windows

Reply:
But OK I think I know what you mean 🙂 . I'm pretty sure issue was that Windows `cmd` doesn't accept std. `--posix` command flags. Not sure about PowerShell but I think that project has been abandoned by Microsoft anyway.

Reply:
Anyway, I tried improving the text in bf396a3. What do you think?

Reply:
By text above I meant the text the page starts with. It's literally mentions "there are some workarounds to solve the performance problems". I don't think we can count using posix as a performance problem. So, we need to make the introduction on that page more general. I would say, let's keep it for the separate ticket as well :)

Reply:
It's just hard to manage and track things where so much stuff is happening in one PR.

Reply:
Totally agree about complicated PR. But in this one case its just a small change which I just did in 57598c3 (: